templates/aws-quicksight.conf

```
@@ -0,0 +1,123 @@
+name = "aws-quicksight"
```

If you look at other template, we don't fill `name` and `code` parameter so it forces customer to set it up before running the scanner.

```suggestion
name = ""
```